api/staged_products_property_collection_helpers_tests.go

```diff
@@ -0,0 +1,283 @@
+package api //not in api_tests because we are intentionally testing some functionality internal to the package
```

Comment:
Same as the other comment, but `collections_service_test.go` (not plural. if you're using a clever IDE like Goland, it won't recognize the auto-open test if a plural as well)

Reply:
Good to know.  Thanks!

Reply:
Addressed in e9c69d6